src/creature.h

```diff
@@ -185,6 +185,11 @@ class Creature
         dealt_projectile_attack projectile_attack( const projectile &proj, const tripoint &target,
                                                    double total_dispersion );
 
+        /**
+         * Return the chance that a projectile attack will hit with at least the given accuracy
```

Comment:
A lot of the project is poorly documented but this is rapidly improving. Its encouraging that you write documentation for your work so if you could write this in the current style that would be great. There are obviously exceptions where a single line is preferable but ambiguous returns are a great example for the use of `@return`